dist/dust-core.js

```diff
@@ -1,6 +1,6 @@
 /*! dustjs-linkedin - v2.7.2
```

Comment:
Just remove the built files from the diff if you would-- we rebuild when we cut a release rather than on every commit.